crates/core/tedge_api/Cargo.toml

```diff
@@ -11,6 +11,7 @@ rust-version = "1.58.1"
 clock = { path = "../../common/clock" }
 csv = "1.1"
 download = { path = "../../common/download" }
+error-chain = "0.12.4"
```

Comment:
This is unused. I also think this is useless here.

Reply:
Fixed